src/Microsoft.Win32.SystemEvents/src/Microsoft/Win32/SystemEvents.cs

```diff
@@ -901,12 +805,12 @@ public static void InvokeOnEventsThread(Delegate method)
         }
 
         /// <internalonly/>
-        /// <devdoc>
-        ///    <para>Kills the timer specified by the given id.</para>
-        /// </devdoc>
+        /// <summary>
+        /// Kills the timer specified by the given id.
+        /// </summary>
```

Comment:
Nit: some of the summary comments have four spaces before the contents and some don't

Reply:
Yeah, I did a find/replace. I can clean it up.